### PR TITLE
RA-1955:Updating Reference Application Order Entry and Allergies Concepts

### PR DESCRIPTION
**Pull request by HerbertYiga**
Updating Reference Application Order Entry and Allergies Concepts

Ticket Id
https://issues.openmrs.org/browse/RA-1955

### CODE REVIEW

**Comment by HerbertYiga** [comment]
cc @dkayiwa  @sherrif10

File: api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-20.xml [hunk 1]
```diff
@@ -31,46 +30,46 @@
...
-<concept concept_id="1513" retired="false"  datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2007-10-21 22:05:03.0" version="" changed_by="1" date_changed="2014-05-16 13:56:54.0" retired_by="1" uuid="1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-<concept concept_id="1514" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2007-10-21 22:08:35.0" version="" changed_by="1" date_changed="2014-05-15 21:01:47.0" retired_by="1" uuid="1514AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-<concept concept_id="1518" retired="false" datatype_id="4" class_id="25" is_set="false" creator="1" date_created="2007-10-21 22:15:30.0" version="" changed_by="1" date_changed="2014-05-15 20:18:17.0" retired_by="1" uuid="1518AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+<concept concept_id="1498" retired="false" form_text="" datatype_id="4" class_id="5" is_set="false" creator="1" date_created="2007-08-29 13:26:20.0" version="" changed_by="1" date_changed="2015-11-05 01:05:41.0" retired_by="1" uuid="1498AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
```
Comment (sherrif10) [inline api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-20.xml:33]: You will need to extract form_text ="" into a separate xml

**Comment by sherrif10** [comment]
Did you forget to add ticket ID

File: api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-20.xml (hunk 1 above)
Comment (HerbertYiga) [reply api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-20.xml:33]: ok then

**Comment by sherrif10** [comment]
@HerbertYiga  if you have exported  a new SQL dump file according to the discussion we had, you need to use a new concept dump files

**Comment by HerbertYiga** [comment]
> if you have exported a new SQL dump file according to the discussion we had, you need to use a new concept dump files

well i am working with the new one

**Comment by HerbertYiga** [comment]
hi @dkayiwa  does this look good enough to be merged in?

**Comment by dkayiwa** [comment]
I find the commit message not clear enough.

**Comment by HerbertYiga** [comment]
> I find the commit message not clear enough.

could it be clear enough now? @dkayiwa

**Comment by tendomart** [comment]
@HerbertYiga  did i miss out seeing the link to the Ticket ?

**Comment by dkayiwa** [comment]
@HerbertYiga did you respond to @sherrif10's comments?

**Comment by HerbertYiga** [comment]
this changes are for a new a new  dump files, the first dump file couldn't generate the package and thats why we picked up on this new one and it gave us this cc @dkayiwa  @sherrif10

**Pull request merged**